packages/npm/@amazeelabs/jest-preset/config/files/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ['@amazeelabs/eslint-config-react'],
```

Comment:
We have to load a different eslint config based on the jest preset. `@amazeelabs/eslint-config-react` will complain about a project without react, so `@amazeelabs/jest-preset` should somehow load `@amazeelabs/eslint-config`.

Reply:
Fixed!